README.md

```diff
@@ -146,7 +182,35 @@ class SomeController {
 
 ### Directive
 
-Static class members of directive controller are used as config directive config.
+You can configure your directive with annotation :
+
+```typescript
+@directive('ngModuleName', {
+    selector:     'atSomeDirective',                //required
+    controllerAs: 'someDirectiveCtrl',              //optional
+    templateUrl:  '/partials/some-directive.html'
+})
+class SomeDirectiveController {
+
+    public static link: angular.IDirectiveLinkFn = (scope, element, attrs, ctrl: SomeDirectiveController) => {
```

Comment:
> I think isn't a best practice to declare a function into an annotation.

Totally agree. That is why I decided to move the configuration to static members, not to annotation.

Problem is that this linking function must be in this example in annotation, or it will not work at all…